### DISCUSSION TITLE
National Student Clearinghouse Reporting Cadence

### DISCUSSION THREAD

**Discussion by MattNickodemus** [ut-effectiveness/national_student_clearinghouse_edify#2] [Q&A] [answered]
@JoyBaule @CSPierce01 @andreabringhurst What should the cadence be around our national student clearinghouse data pulls and reporting? I know that IE uses this data for several reports, and that we have other requirements that this process depends upon. More specifically,

- When should we submit our data to NSC?
- When do we have the NSC data available in Edify?
- When will we deliver the reports on melt, transfer, etc?

For each of these questions, let's also lay out the reasons for the timetables.

**Comment by andreabringhurst** [comment]
Much of our timing is driven by the National Student Clearinghouse. Our data is ready after census (typically early September). But not every institution has similar census dates. IPEDS allows census dates to be as late as Oct. 15. Which means that National Student Clearinghouse data is not all submitted until late October. This means that we should not submit our data to National Student Clearinghouse until early November. Waiting this long gives us better data. Turn around time from National Student Clearinghouse is 3-5 days. Then we would need time to upload data into our systems and time to generate reports.

- We should submit data to National Student Clearinghouse in early November.
- We would be able to get our data in Edify around mid-late November.
- We would be able to produce reports in early December. 

That would give our office 4-6 weeks to complete these reports while also competing other required tasks.

**Reply by MattNickodemus** [reply]
What is our spring schedule? @CSPierce01 Does this match the schedule that we currently have?

**Reply by CandiPierce** [reply]
I have the NSCH uploads scheduled for November 1 and April 1. If you prefer, we could move spring to March 15.

**Reply by MattNickodemus** [reply]
This is really up to you. We just need to set a schedule and then stick to it.

**Reply by CandiPierce** [reply, marked as answer]
@MattNickodemus we will upload files to NSCH the first week of November and the third week of March.
